.gitignore

```diff
@@ -106,4 +106,8 @@ ENV/
 
 # End of https://www.gitignore.io/api/python
 old/
-test_credentials.json
\ No newline at end of file
+test_credentials.json
+
+# IDEs
+.idea
+.vscode
```

Comment:
Macht es ggf. Sinn, die config.json hier auch noch zu ignoren?

Reply:
Könnte man machen, ich sehe es aber nicht wirklich als notwendig an. Wir schlagen in unseren Beispielen ja immer nur vor, dass die Datei so heißen könnte (man übergibt ja den Pfad an das CLI). Man benötigt sie jedoch grundsätzlich nicht, könnte sie auch anders nennen oder auch an einem völlig anderen Ort liegen haben (z.B. außerhalb des Repos), weswegen ich nicht glaube, dass es viel hilft, eine der unendlich vielen Möglichkeiten zum Definieren dieser Datei hier auszuschließen.